datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVHelper.scala

```diff
@@ -463,6 +485,23 @@ object MVHelper {
     }
   }
 
+  // Create the aliases using two plan outputs mappings.
+  def createAliases(mappings: Seq[(NamedExpression, NamedExpression)]): Seq[NamedExpression] = {
+    val oList = for ((o1, o2) <- mappings) yield {
```

Comment:
it is better to use `map` instead of `for`

Reply:
ok